contract-addresses/dependencies.ts

```diff
@@ -1,5 +1,735 @@
+    contractDependencies: ['core', 'tribeUsdCompositeOracle', 'tribeMinter', 'collateralizationOracleWrapper', 'tribe']
+  },
+  aaveEthPCVDeposit: {
+    contractDependencies: ['core', 'aaveEthPCVDripController', 'bondingCurve', 'pcvGuardian']
+  },
+  aaveFeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  aaveRaiPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  agEurAngleUniswapPCVDeposit: {
+    contractDependencies: ['core', 'fei', 'chainlinkEurUsdOracleWrapper']
+  },
+  balancerDepositBalWeth: {
+    contractDependencies: ['core', 'balUsdCompositeOracle', 'chainlinkEthUsdOracleWrapper']
+  },
+  compoundDaiPCVDeposit: {
+    contractDependencies: ['core', 'daiBondingCurve', 'daiPCVDripController', 'daiPSM']
+  },
+  compoundEthPCVDeposit: {
+    contractDependencies: ['core', 'bondingCurve', 'compoundEthPCVDripController', 'pcvGuardian']
+  },
+  d3poolConvexPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  d3poolCurvePCVDeposit: {
+    contractDependencies: ['core']
+  },
+  dpiUniswapPCVDeposit: {
+    contractDependencies: ['core', 'fei', 'chainlinkDpiUsdOracleWrapper']
+  },
+  ethLidoPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  ethTokemakPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  feiLusdLBPSwapper: {
+    contractDependencies: ['core', 'chainlinkLUSDOracleWrapper']
+  },
+  indexCoopFusePoolDpiPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  indexCoopFusePoolFeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  indexDelegator: {
+    contractDependencies: ['core']
+  },
+  liquityFusePoolLusdPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  poolPartyFeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool18FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool19DpiPCVDeposit: {
+    contractDependencies: ['core']
+  },
+  rariPool19FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool22FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool24FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool25FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool26FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool27FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool28FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool31FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool54FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool6FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool72FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool79FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool7FeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
+  },
+  rariPool7LusdPCVDeposit: {
+    contractDependencies: ['core']
```

Comment:
missing lusd, and the corresponding fToken

Reply:
both external